src/main/resources/edu/cornell/kfs/module/purap/businessobject/datadictionary/JaggaerRoleLinkMapping.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:p="http://www.springframework.org/schema/p"
+       xsi:schemaLocation="http://www.springframework.org/schema/beans
+                http://www.springframework.org/schema/beans/spring-beans.xsd">
+
+    <bean id="JaggaerRoleLinkMapping" parent="JaggaerRoleLinkMapping-parentBean"/>
+    <bean id="JaggaerRoleLinkMapping-parentBean"
+          abstract="true"
+          parent="BusinessObjectEntry"
+          p:actionsProvider-ref="businessObjectActionsProvider"
+          p:businessObjectAdminService-ref="defaultBoAdminService"
+          p:businessObjectClass="edu.cornell.kfs.module.purap.businessobject.JaggaerRoleLinkMapping"
+          p:inquiryDefinition-ref="JaggaerRoleLinkMapping-inquiryDefinition"
+          p:lookupDefinition-ref="JaggaerRoleLinkMapping-lookupDefinition"
+          p:name="JaggaerRoleLinkMapping"
+          p:objectLabel="Jaggaer Role Link Mapping"
+          p:searchService-ref="defaultSearchService"
+          p:titleAttribute="jaggaerRoleName">
+        <property name="keyAttributes">
+            <list>
+                <ref bean="JaggaerRoleLinkMapping-jaggaerRoleName"/>
+            </list>
+        </property>
+        <property name="attributes">
+            <list>
+                <ref bean="JaggaerRoleLinkMapping-jaggaerRoleName"/>
+                <ref bean="JaggaerRoleLinkMapping-eShopLink"/>
+                <ref bean="JaggaerRoleLinkMapping-contractsPlusLink"/>
+                <ref bean="JaggaerRoleLinkMapping-jaggaerAdminLink"/>
+                <ref bean="JaggaerRoleLinkMapping-active"/>
+            </list>
+        </property>
+    </bean>
+
+    <bean id="JaggaerRoleLinkMapping-jaggaerRoleName" parent="JaggaerRoleLinkMapping-jaggaerRoleName-parentBean"/>
+    <bean id="JaggaerRoleLinkMapping-jaggaerRoleName-parentBean"
+          abstract="true"
+          parent="AttributeDefinition"
+          p:name="jaggaerRoleName"
+          p:forceUppercase="false"
+          p:label="Jaggaer Role Name"
+          p:shortLabel="Jaggaer Role"
+          p:maxLength="50"
+          p:validationPattern-ref="AnyCharacterWithWhitespaceValidation">
+        <property name="control">
+            <bean parent="TextControlDefinition" p:size="52"/>
+        </property>
+    </bean>
+
+    <bean id="JaggaerRoleLinkMapping-eShopLink" parent="JaggaerRoleLinkMapping-eShopLink-parentBean"/>
+    <bean id="JaggaerRoleLinkMapping-eShopLink-parentBean"
+          abstract="true"
+          parent="GenericAttributes-activeIndicator"
+          p:name="eShopLink"
+          p:label="Assignable on eShop Login Link"
```

Comment:
I'm not sure about "Assignable" here.  What about "Available" on eShop link?

Reply:
I'll consult with the functionals on the naming like you suggested.